### DISCUSSION TITLE
DICOM conversion

### DISCUSSION THREAD

**Discussion by loneblues** [QSMxT/QSMxT#66] [Q&A] [answered]
Hello. 
Thanks for the amazing software. 
I have been trying to use QSMxt for my data, but dicomConvert.py doesn't seem to quit act as intended, since only nifti files named ~~mag is generated, and only json file for 1st echo and no nifti is saved. Also, when opening these nifti files, they are phase data. 
My guess is that the dicoms need to be arranged in a certain order. 
Is there a way to make this work, other than manually coverting to nifti, then starting from there? 

Thanks in advance. 

Best, 
HJ Lee

P.S. Another thing is that we have a ")" in our protocol description and this results in an error message while using the BIDS conversion script.

**Comment by stebo85** [comment]
Dear @loneblues, 

can you send the dicom data to me (s.bollmann@uq.edu.au) so I can have a look?

Thank you
Steffen

**Reply by loneblues** [reply]
Dear Steffen, 

Of Course, but i need to get it anonymized, first. May take a few hours, since I have left my office, just now. 

Since I posted the question, I also found that the dicom convert script changed the the phase file names to the same file names as the magnitude data, so I think that's where all the trouble started. I tried to change the file names to have _mag and _phase patterns, but qsm processing kept printing errors, and based on the logs I presume that the ")" that we have in our protocol description is causing all these problems. 

As said, will try to send the DICOMs as fast as possible. 

Thanks! 

Best 
HJ lee

**Reply by loneblues** [reply]
Dear Steffen, 

I have sent you the DICOM files yesterday. 
Please let me know if it didn't arrive in your mail. 

Thanks. 

Sincerely, 
HJ Lee

**Reply by stebo85** [reply]
Dear @loneblues ,

Your data arrived and we are currently working on it. It has a couple of issues, but it should work :)

Cheers
Steffen

**Comment by stebo85** [comment, marked as answer]
Quick summary of the solution: The phase data was interpolated and this led to artifacts around phase wraps -> recommended to not interpolate or use complex data -> this data then looked good, but requires larger field of view. Sorting of data is now fixed and was broken by a bracket in the protocol name.
